.openzeppelin/project.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What is this configuration used for?

Reply:
This is the configuration of your project. Usually this is where we put the proxies addresses and in our case the implementations for the sake of transparency, is the OZ standard, now, we must see if it's compatible with our customproxy